src/grisp_bitmap.erl

```diff
@@ -1,3 +1,7 @@
+%% -----------------------------------------------------------------------------
```

Comment:
Can we remove these dividers? I don't think they're necessary at the start of the file

Reply:
yes.